src/components/button.jsx

```diff
@@ -0,0 +1,17 @@
+import React, {Component} from 'react'
+
+class Button extends Component {
```

Comment:
Should this have a prop for the name on the button? That way, we can customize the buttons to have specific wording on them.

Reply:
Yup, I need implement mapStateToProps from parent obj's soon.